src/background.js

```diff
@@ -0,0 +1,2 @@
+browser.blurts.start();
+browser.blurts.onTelemetryEvent.addListener(console.log);
```

Comment:
I imagine this will need to change to something like:
```
const studySetup = {...} // See https://github.com/mozilla/blurts-addon/blob/b3c6146c70da552ae13fc07629f7b0997ac334f1/background.js
const studyInfo = browser.study.setup(studySetup);
browser.blurts.start(studyInfo); // To pass studyInfo.variation to the pop-up
browser.blurts.onTelemetryEvent.addListener(browser.study.sendTelemetry);
```